src/github.com/getlantern/geolookup/geolookup.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const (
 	geoServeEndpoint = `http://geo.getiantem.org/lookup/%s`
-	geoLookupTimeout = 20 * time.Second
+	geoLookupTimeout = 60 * time.Second
```

Comment:
Maybe @myleshorton's suggestion of pushing this to 600 is a bit risky, let's try with a much smaller increment for now and see how does it behave.